tests/kafkatest/services/console_consumer.py

```diff
@@ -119,6 +123,22 @@ def __init__(self, context, num_nodes, kafka, topic, message_validator=None, fro
         self.message_validator = message_validator
         self.messages_consumed = {idx: [] for idx in range(1, num_nodes + 1)}
 
+        # Process client configuration
```

Comment:
@guozhangwang I think the change suggested by @granders was to remove the if-then-else around render() which was in the code earlier. That was removed in the latest patch (the code selection above is slightly confusing). Can you let me know if I have misunderstood the comment?